dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFields.java

```diff
@@ -279,11 +307,23 @@ public String getAllelesInForwardStrand() {
                 ^ this.getContigOrientation().equals(Orientation.FORWARD);
 
         String alleles = this.getAlleles();
-        if (forward) {
-            return alleles;
-        } else {
-            return calculateReverseComplement(alleles);
+        String forwardAlleles = forward ? alleles : calculateReverseComplement(alleles);
+
+        String[] splitAlleles = forwardAlleles.split("/", -1);
+        return Stream.of(splitAlleles).map(this::getTrimmedAllele).collect(Collectors.joining("/"));
```

Comment:
Why this? Are there allele strings like "A/T /G"? (containing spaces)

Reply:
yes :( look at dbsnp_150, obsvariation, var_id = 72. var_id=63 is also funny (`C/T/`)